assets/schema/ensemble_schema.json

```diff
@@ -2812,8 +2812,15 @@
                       "type": "string",
                       "description": "Where to display the indicator if specified",
                       "enum": [
```

Comment:
don't hardcode this. There must be an alignment schema that we declare already. If not then create it.

Reply:
Done